resources/reconciler/charts/component-reconcilers/values.yaml

```diff
@@ -0,0 +1,11 @@
+# Default values for component-reconcilers.
+
+nameOverride: ""
+fullnameOverride: ""
+
+components:
+- istio
```

Comment:
Maybe better to fail if the list is empty? Otherwise we should to update this list when ading/removing new components to stay in-sync